<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>924</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>7</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Conda Environment:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="combo_envs">
      <property name="font">
       <font>
        <pointsize>7</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Slect conda environment with PCRaster, GDAL and other libs installed</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QToolButton" name="toolButton">
      <property name="font">
       <font>
        <pointsize>7</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources/lisemdbase.qrc">
        <normaloff>:/2X/question-mark-button2x.png</normaloff>:/2X/question-mark-button2x.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>7</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Python dbase script</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_Script">
      <property name="font">
       <font>
        <pointsize>7</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QToolButton" name="toolButton_script">
      <property name="font">
       <font>
        <pointsize>7</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources/lisemdbase.qrc">
        <normaloff>:/2X/Folder-Open-icon.png</normaloff>:/2X/Folder-Open-icon.png</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_Maps">
      <property name="font">
       <font>
        <pointsize>7</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Maps and Folders</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <property name="horizontalSpacing">
        <number>8</number>
       </property>
       <property name="verticalSpacing">
        <number>4</number>
       </property>
       <item row="7" column="7">
        <widget class="QToolButton" name="toolButton_LULC">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources/lisemdbase.qrc">
           <normaloff>:/2X/Folder-icon.png</normaloff>:/2X/Folder-icon.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="5">
        <widget class="QLineEdit" name="lineEdit_Base">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="5" colspan="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>412</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="6">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" rowspan="2" colspan="3">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Base DEM (reference map)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="7" rowspan="2">
        <widget class="QToolButton" name="toolButton_base">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources/lisemdbase.qrc">
           <normaloff>:/2X/Folder-icon.png</normaloff>:/2X/Folder-icon.png</iconset>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_17">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>LULC map (tif)</string>
         </property>
        </widget>
       </item>
       <item row="9" column="4">
        <widget class="QToolButton" name="toolButton_LULCTable">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources/lisemdbase.qrc">
           <normaloff>:/2X/Folder-Open-icon.png</normaloff>:/2X/Folder-Open-icon.png</iconset>
         </property>
        </widget>
       </item>
       <item row="9" column="6">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="5" rowspan="2" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>412</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Maps openLISEM database folder</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="5">
        <widget class="QLineEdit" name="lineEdit_Maps">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Base data input folder</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="4">
        <widget class="QToolButton" name="toolButton_LULCMap">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources/lisemdbase.qrc">
           <normaloff>:/2X/Folder-Open-icon.png</normaloff>:/2X/Folder-Open-icon.png</iconset>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QLineEdit" name="lineEdit_LULCTable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QLineEdit" name="lineEdit_LULCMap">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Base Channel (reference map)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="7" rowspan="2">
        <widget class="QToolButton" name="toolButton_maps">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources/lisemdbase.qrc">
           <normaloff>:/2X/Folder-icon.png</normaloff>:/2X/Folder-icon.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="lineEdit_baseDEM">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="7" column="2" colspan="5">
        <widget class="QLineEdit" name="lineEdit_LULC">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QToolButton" name="toolButton_baseChannel">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources/lisemdbase.qrc">
           <normaloff>:/2X/Folder-Open-icon.png</normaloff>:/2X/Folder-Open-icon.png</iconset>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>LULC table</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QToolButton" name="toolButton_baseDEM">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources/lisemdbase.qrc">
           <normaloff>:/2X/Folder-Open-icon.png</normaloff>:/2X/Folder-Open-icon.png</iconset>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>LULC map (tif)</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="lineEdit_baseChannel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_start">
      <property name="text">
       <string>Create Database</string>
      </property>
      <property name="icon">
       <iconset resource="resources/lisemdbase.qrc">
        <normaloff>:/2X/play-icon.png</normaloff>:/2X/play-icon.png</iconset>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QToolButton" name="toolButton_clear">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources/lisemdbase.qrc">
        <normaloff>:/2X/noscreen.png</normaloff>:/2X/noscreen.png</iconset>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Progress:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="text_out"/>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_Options">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Soil Layer 1 </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Soil Layer 2 </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QCheckBox" name="checkBox_LULC">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Include Land Use / Land Cover</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_SGlayer1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="5">
        <widget class="QCheckBox" name="checkBox_DEM">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Include DEM derivatives (slope, LDD network)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="3">
        <widget class="QComboBox" name="comboBox_SGlayer2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="6">
        <widget class="QCheckBox" name="checkBox_Soilgrids">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Include download SOILGRIDS data (texture data will be stored in the base folder)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QToolButton" name="toolButton_CheckAll">
         <property name="text">
          <string>V</string>
         </property>
         <property name="icon">
          <iconset resource="resources/lisemdbase.qrc">
           <normaloff>:/2X/reset.png</normaloff>:/2X/reset.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QCheckBox" name="checkBox_Channels">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Include Channels</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="5">
        <widget class="QCheckBox" name="checkBox_Infil">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Include Infiltration parameters</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Config_file">
   <property name="text">
    <string>Load Config file</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/lisemdbase.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBox_LULC</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_LULC</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_LULC</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_9</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_LULC</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_LULCTable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_LULC</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButton_LULCTable</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>885</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_LULC</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButton_LULC</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>891</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
